### ISSUE TITLE
Are there any sample codes of the combining of dragging demos?

### ISSUE THREAD

**Issue by elva329** [atlassian/react-beautiful-dnd#2181]
Hi,

I found the combining dragging behavior demo on this link (https://react-beautiful-dnd.netlify.app/?path=/story/board--with-combining), by reading your document, I got to know some basic information about combining dragging, however, there still some uncertainties that I want to confirm.

I am wondering if If there are some sample codes of the combining dragging demos. It will be much appreciated if you could provide some reference code.

Thanks a lot.

**Comment by 2stash** [comment]
@elva329 Hi. It sounds like you did not find the actual code for the demo? If that is the case, then the link is below. 

https://github.com/atlassian/react-beautiful-dnd/blob/master/stories/src/board/board.jsx


When you start looking through the board.jsx code, remember the links to other components is relative to the path below.
react-beautiful-dnd/stories/src/board/board.jsx /

Or were you just looking for more examples?

**Comment by elva329** [comment]
Hi 2stash,

Thanks for your reply, the link you sent is very useful. I just have another question, is there a way that I can drag a parent container to another parent container ?

Thank you.

Regards,

**Comment by 2stash** [comment]
So like if you have 3 columns, and in each column you had Todos that could be dragged to each column, you want to drag a column into another column?

**Comment by elva329** [comment]
Hi,

 I attached a screenshot of my expected behaviour, I want to drag Draggable1 which is a parent container to Draggable2 (a parent container, too) to make it a child item of Draggable2? Is it feasible in react-beautiful-dnd ?

![image](https://user-images.githubusercontent.com/15085514/115251297-fe461100-a15c-11eb-91ae-4387ec1ed337.png)

Thank you.

Regards,

**Comment by 2stash** [comment]
Yes you can do that. I am searching for a working example, I have one half created I will finish in a day or so when I get some time.

I did find this example below, which is using the Atlassian Tree. I haven't used their Tree, but I am going to dig into it more. It looks better suited for when your draggables are just a simple name, like lists, folders, or table of contents.

https://codesandbox.io/s/v6m657wk70?fontsize=14
